src/main/webapp/components/workspace/hooks/use-save-workspace.tsx

```diff
@@ -28,7 +32,7 @@ export default () => {
           id,
           attrs: {
             ...workspace,
-            queries,
+            ...(workspace.queries && { queries }),
```

Comment:
:pencil2: I think you could move this logic above:
`const queries = workspace.queries && workspace.queries.map(.....`